crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -56,6 +57,10 @@ pub struct AuthorityPerEpochStore {
     consensus_notify_read: NotifyRead<ConsensusTransactionKey, ()>,
     epoch_alive: NotifyOnce,
     end_of_publish: Mutex<StakeAggregator<(), true>>,
+    /// Pending certificates that we are waiting to be sequenced by consensus
+    /// Lock ordering: this is a 'leaf' lock, no other locks should be acquired in the scope of this lock
+    /// In particular, this lock is always acquired after taking read or write lock on reconfig state
+    pending_consensus_certificates: Mutex<HashSet<TransactionDigest>>,
```

Comment:
Can we mention that the digests in the set should match with user transactions in `pending_consensus_transactions`, and this is only used for checking if there is any inflight consensus transaction from users?

Reply:
Added more details to the description